### PR TITLE
Adjusted pull request Swedish holidays (#44) by @johanpalmqvist according to Swedish holiday laws and comments.

### PR DESCRIPTION
**Pull request by karl-henrik**
Because nothing had been done for pull request #44 since late March, I have made changes according to the comments. I have also reviewed the holidays and removed days that are work free according to Swedish vacation laws but not actual holidays according to holiday law [1989-253](http://www.riksdagen.se/sv/dokument-lagar/dokument/svensk-forfattningssamling/lag-1989253-om-allmanna-helgdagar_sfs-1989-253)

### CODE REVIEW

File: holidays.py [hunk 1]
```diff
@@ -2225,3 +2225,101 @@ def _populate(self, year):
...
+        if year <= 2004:
+            self[pentecost_day_two] = "Annandag pingst"
+
+        # Midsummer day. Saturday between June 20th and June 26th
+        if year >= 1953:
+            self[date(year, 6, 20) + rd(weekday=SA)] = "Midsommardagen"
+        else:
+            self[date(year, 6, 24)] = "Midsommardagen"
+        # All saints day. Friday between October 31th and November 6th
+        self[date(year, 10, 31) + rd(weekday=SA)] = "Alla helgons dag"
```
Comment (dersger) [inline holidays.py]: Alla helgons dag was not a holiday before 1953  (changed by law SFS 1952:48).
Also change comment to say "Saturday".

File: holidays.py [hunk 2]
```diff
@@ -2225,3 +2225,101 @@ def _populate(self, year):
...
+
+        if year <= 1953:
+            self[date(year, 3, 25)] = "Jungfru Marie bebådelsedag"
```
Comment (dersger) [inline holidays.py]: Jungfru Marie bebådelsedag should be before 1953 (i.e. not including 1953).

**Comment by dersger** [comment]
Update README.rst as well (as in #44)

**Comment by karl-henrik** [comment]
Are there something more to this or isn't it ready to merge?

**Pull request merged**